src/renderer/components/rating/index.tsx

```diff
@@ -16,17 +13,37 @@ const StyledRating = styled(MantineRating)`
     }
 `;
 
-export const Rating = ({ onClick, ...props }: RatingProps) => {
-    // const debouncedOnClick = debounce(onClick, 100);
+export const Rating = ({ onChange, ...props }: RatingProps) => {
+    const valueChange = useCallback(
+        (rating: number) => {
+            if (onChange) {
+                if (rating === props.value) {
+                    onChange(0);
+                } else {
+                    onChange(rating);
+                }
+            }
+        },
+        [onChange, props.value],
+    );
+
+    const debouncedOnChange = debounce(valueChange, 100);
 
     return (
         <Tooltip
-            label="Double click to clear"
+            label="Click on the same value to clear"
```

Comment:
Label can be removed now since this is the typical functionality for the rating component. I only added the label previously because double click is non-standard.